instruqt-tracks/consul-life-of-a-developer/01-service-mesh-environment-review/assignment.md

```diff
@@ -1,6 +1,6 @@
 ---
 slug: service-mesh-environment-review
-id: 8nmk95wj6ftw
+id: rg5oxrbqewzh
```

Comment:
When IDs change, it usually breaks the Instruqt track tests in Circle CI.  If the IDs stay the same or are stripped completely, everything is fine.  Can you either add a commit to strip the IDs throughout the assignments or replace their original values?

Reply:
Instruqt attempts to do conflict detection on track upload -- the CircleCI tests begin by uploading the track content before every nightly test so if Instruqt detects a conflict on an upload to an existing track and stops the track upload, the actual test doesn't get executed.  In future when we move to GitHub Actions and change how the nightly tests and other track-related actions work, this hopefully won't break the nightly tests since they'll be separate from updating track content.